docs/News.md

```diff
@@ -1,3 +1,9 @@
+## August 13, 2019
+*   **Added data** consisting of 295 samples from 3 studies:
+    *  [Pediatric Preclinical Testing Consortium (PPTC, 2019)](https://www.cbioportal.org/study/summary?id=pptc_2019) *261 samples*
```

Comment:
Just an FYI it says (Maris, 2019) on Study View.

Reply:
Ya JJ suggested to change it to PPTC, Yichao changed it in the database but it's not reflecting yet.